PubNubUnity/Assets/PubNub/Helpers/TokenHelpers.cs

```diff
@@ -0,0 +1,411 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using PeterO.Cbor;
+using System.Text;
+
+
+namespace PubNubAPI
+{
+    public enum PNGrantBitMask
+    {
+        PNRead = 1,
+        PNWrite = 2,
+        PNManage = 4,
+        PNDelete = 8,
+        PNCreate =16,
```

Comment:
Need to check with Rai on create. When there is no permission flag for Create, why do we need permission mapping?

Reply:
backward compatibility, thats why it is there is other places as well